test/redislist.js

```diff
@@ -3,7 +3,29 @@
 var assert = require('assert'),
     mockery = require('mockery');
 
-mockery.registerSubstitute('redis', 'redis-mock');
+const { inherits } = require('util');
+
+const commands = require('ioredis-mock/lib/commands');
+const commandsCopy = Object.assign({}, commands);
+let subscriber;
+
+class RedisMockExtended {
+    brpop (subscriptionPath, body, queueFn) {
+        queueFn(undefined, [1, 2]);
+    }
+
+    lpush(subscriptionPath, body, queueFn) {
+        queueFn(undefined);
+    }
+}
+
+mockery.registerMock('./commands', commandsCopy);
+
+const RedisMock = require('ioredis-mock');
```

Comment:
Can we put this on top? It would be neater to have all the requirements on top of the file (sorry to be picky :) )